src/main/java/org/cryptomator/ui/mainwindow/VaultListController.java

```diff
@@ -101,6 +104,7 @@ public class VaultListController implements FxController {
 		this.appWindows = appWindows;
 
 		this.emptyVaultList = Bindings.isEmpty(vaults);
+		this.useCondensedMode = Bindings.createBooleanBinding(settings.useCondensedMode::get, settings.useCondensedMode);
```

Comment:
Instead of ceating this duplicate binding, create directly the double binding in line 116ff here in the constructor and use the settings.useCondensedMode property directly.